pkg/examples/sleep.go

```diff
@@ -28,13 +40,21 @@ type Sleep struct {
 
 func (s *Sleep) Install() error {
 	util.Log.Infof("Deploying Sleep in namespace %s", s.Namespace)
+	proxy, _ := util.GetProxy()
+	configmap := util.RunTemplate(sleepConfigmap, proxy)
+	util.Log.Infof("Creating configmap %s", configmap)
+	util.KubeApplyContents(s.Namespace, configmap)
```

Comment:
@fjglira Please add an uninstall call in the Uninstall method below for the new configmap.

Reply:
Added